core/src/Telemetry/DefaultEvent.cs

```diff
@@ -25,18 +25,35 @@
 //
 //------------------------------------------------------------------------------
 
-using Microsoft.Identity.Core;
+using System.Collections.Generic;
+using System.Globalization;
 
 namespace Microsoft.Identity.Core.Telemetry
 {
     internal class DefaultEvent : EventBase
     {
-        public DefaultEvent(string clientId) : base(EventNamePrefix + "default_event")
+        public DefaultEvent(string clientId, IDictionary<string, int> eventCount) : base((string) (EventNamePrefix + "default_event"))
         {
             var platformProxy = PlatformProxyFactory.GetPlatformProxy();
             this[EventNamePrefix + "client_id"] = clientId;
             this[EventNamePrefix + "sdk_platform"] = platformProxy.GetProductName()?.ToLowerInvariant();
             this[EventNamePrefix + "sdk_version"] = MsalIdHelper.GetMsalVersion();
+            this[EventNamePrefix + "application_name"] = PlatformProxyFactory.GetPlatformProxy().GetCallingAssemblyName()?.ToLowerInvariant();
```

Comment:
>PlatformProxyFactory.GetPlatformProxy() [](start = 57, length = 39)

pls. use the platformProxy variable from above, instead of calling the factory.

Reply:
done